packages/owl-bot/src/owl-bot.ts

```diff
@@ -382,40 +382,29 @@ async function handlePullRequestLabeled(
     return;
   }
 
-  const command_labels = [OWLBOT_RUN_LABEL, OWL_BOT_COPY_COMMAND_LABEL];
-
-  // Only run on label event if the label added is the owlbot label
-  if (!command_labels.includes(payload.label.name)) {
-    logger.info(
-      `skipping non-owlbot label: ${payload.label.name} ${head} for ${base}`
+  const commandLabels = [OWLBOT_RUN_LABEL, OWL_BOT_COPY_COMMAND_LABEL];
+
+  if (payload.pull_request.draft && payload.label.name === OWL_BOT_COPY) {
```

Comment:
nit, the `noop` is a bit confusing, I might change this logic around a bit so the "non-owlbot" case breaks, rather than the happy path doing nothing.